plugins/include/amxmisc.inc

```diff
@@ -26,7 +26,7 @@
 stock is_user_admin(id)
 {
 	new __flags = get_user_flags(id);
-	return (__flags > 0 && !(__flags & ADMIN_USER));
+	return (__flags > ADMIN_ALL && !(__flags & ADMIN_USER));
```

Comment:
Should it logically be `!= ADMIN_ALL`?

Reply:
no difference ? 

__flags can't go less then 0 at least here.

Reply:
What if 32-nd bit is set? This is undocumented but can be.

Reply:
admin flags are hardcoded in the default AmxModX admin system also i don't think that we can change this because of backward compatibility

Reply:
Well `> 0` is just here to say "if the value is positive". 

It's also fine to use `ADMIN_ALL`, but it should be logically  `!= ADMIN_ALL` as stated WPMGPRoSToTeMa.

Reply:
updated